.azure-pipelines/other/disabled_tests.txt

```diff
@@ -1,3 +1,4 @@
+tests/basic/pthread_join/Makefile
```

Comment:
This test is disabled until https://github.com/lsds/sgx-lkl/issues/686 is fixed.